docs/source/enterprise-guide/change-management.mdx

```diff
@@ -0,0 +1,543 @@
+---
+title: Change management in Apollo Federation
+sidebar_title: Change management
+---
+
+import { ExpansionPanel } from "gatsby-theme-apollo-docs/src/components/expansion-panel";
+
+A federated GraphQL API is a multi-tiered architecture made up of independent
+components. GraphQL requests flow through multiple components to construct a
+response, which in turn flow back through those same components to create a
+response, as illustrated below:
+
+```mermaid
+graph LR
+    X(Client) --> A(Gateway)
+    A -.-> X
+    A --> B(Subgraph A)
+    B -.-> A
+    A --> C(Subgraph B)
+    C -.-> A
+    B --> D(REST API)
+    D -.-> B
+    D --> E(Database)
+    E -.-> D
+    C --> F(gRPC API)
+    F -.-> C
+    F --> G(Third party API)
+    G -.-> F
+```
+
+Allowing teams to independently update the components they own is a key benefit
+of federation. However, understanding and de-risking changes to a component can
+be challenging, especially because changes might require cascading changes in
+other components. This is especially true for GraphQL schema definitions, which
+must be coordinated between subgraph services and the gateway.
+
+This section covers the most common change management scenarios in a federated
+architecture. We'll consider the proper sequencing of release events, how Apollo
+tooling makes managing changes easier, and when progressive delivery techniques
+like blue-green deploys are useful.
+
+Specifically, we'll examine scenarios that:
+
+- Involve a schema change
+  - [Backward compatible subgraph schema changes](#backward-compatible-subgraph-schema-changes)
+  - [Backward _incompatible_ subgraph schema changes](#backward-incompatible-subgraph-schema-changes)
+- Involve schema changes in multiple subgraphs
+  - [Changes to shared types](#changes-to-shared-types)
+  - [Changes to query planning](#changes-to-query-planning-field-migrations-entity-ownership-changes-etc)
+- Don't involve schema changes
+  - [Changes to a data source](#changes-to-a-data-source)
+  - [Changes to subgraph resolvers](#changes-to-subgraph-resolvers)
+  - [Changes to gateway logic/configuration](#changes-to-gateway-logic-or-configuration)
+
+The release guidelines for each scenario should generally apply to all release
```

Comment:
I think we should also clearly state here that all of the release workflows below assume the use of managed federation with Apollo Studio's schema registry.